src/main/java/cholog/supp/api/post/service/PostService.java

```diff
@@ -77,7 +77,6 @@ public EachPostResponse getEachPost(Member member, Long postId) {
         return new EachPostResponse(eachPost, comments);
     }
 
-    @Transactional(readOnly = true)
     public List<EachComment> getCommentList(Long studyId, List<Comment> comments, Member member) {
         return comments.stream().map(comment -> {
             Member nowCommentMember = comment.getMember();
```

Comment:
```suggestion
            Member commentMember = comment.getMember();
```
변수 네이밍을 이렇게 바꾸는건 어때?? `nowCommentMember`도 직관적인 표현이긴 한데, now가 없어도 이해가 되는 의미라고 생각해서!
편한대로 적용하면 좋을거 같어!

Reply:
<img width="976" alt="스크린샷 2024-10-22 오후 5 34 54" src="https://github.com/user-attachments/assets/cf795a2b-c5e7-4fe3-915a-85590a4e8f87">
<img width="976" alt="스크린샷 2024-10-22 오후 5 35 39" src="https://github.com/user-attachments/assets/bbd658eb-f63a-4d2f-9cd8-afa4584d09be">
두 이미지를 보면 변수 네이밍에 대해 깃헙 코드로 해본건데, `nowCommentMember` 변수 네이밍이 우리 코드밖에 없네😂